service-registry/src/test/java/io/servicecomb/serviceregistry/version/TestVersionRuleLatestParser.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.serviceregistry.version;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import io.servicecomb.serviceregistry.definition.DefinitionConst;
+
+public class TestVersionRuleLatestParser {
+  VersionRuleParser parser = new VersionRuleLatestParser();
+
+  VersionRule versionRule = parser.parse(DefinitionConst.VERSION_RULE_LATEST);
```

Comment:
How can we tell the latest version? It looks like very version is OK.

Reply:
firstly, we must have a version set, and then we can determine which one is the latest.
"latest" can not be a simple rule, it must work with data.

i have changed the implementation